activerecord/lib/active_record/touch_later.rb

```diff
@@ -0,0 +1,36 @@
+module ActiveRecord
+  # = Active Record Touch Later
+  module TouchLater
+    extend ActiveSupport::Concern
+
+    included do
+      before_commit_without_transaction_enrollment :touch_defered_attributes
+    end
+
+    def touch_later(*names) # :nodoc:
+      raise ActiveRecordError, "cannot touch on a new record object" unless persisted?
+
+      @_defer_touch_attrs ||= timestamp_attributes_for_update_in_model
+      @_defer_touch_attrs |= names
+      @_touch_time = current_time_from_proper_timezone
+
+      @_defer_touch_attrs.each { |attr| write_attribute attr, @_touch_time }
+      clear_attribute_changes @_defer_touch_attrs
```

Comment:
Would keep `#surreptitiously_touch` to encapsulate this bit of trickery. Easier to understand